javascript/javascript1/week3/notes-app.js

```diff
@@ -0,0 +1,62 @@
+const notes = [];
+
+// Function for saving note
+function saveNote(content, id) {
+    notes.push({ "content": content, "id": id });  // push object to notes array 
+    return notes;
+}
+
+saveNote("Pick up groceries", 1);
+saveNote("Do laundry", 2);
+saveNote("Wash dishes", 3);
+
+console.log(notes);
+
+
+
+// Function for getting a specific note
+function getNote(id) {
+
+    if (!id || typeof id !== "number") {
+        return "An error string";
```

Comment:
Small but widely used trick: 
code below does the same, but little bit "simpler"
```javascript
if (!id || typeof id !== "number") {
   return "An error string";
}
for (let i = 0; i < notes.length; i++) {
    if (notes[i].id === id) {
        return notes[i];
    }
}
return "The id is not in any object";
```